### DISCUSSION TITLE
reCAPTCHA在国内访问不了，可以添加能在国内访问的hCaptcha吗？

### DISCUSSION THREAD

**Discussion by MrZhang365** [walinejs/waline#1710] [Ideas]
您好，  
最近我发现reCAPTCHA在国内访问不了。但是hCaptcha和reCAPTCHA差不多，唯一的区别是hCaptcha可以在国内访问，可以添加hCaptcha支持吗？

**Comment by lizheming** [comment]
https://www.17ce.com/site/http/20230102_721138b08a8811eda16def0f096a0469:1.html reCAPTCHA 国内可以访问呀，hCaptcha 欢迎 PR

**Reply by MrZhang365** [reply]
啊我说的是谷歌官方的，我无法上谷歌申请key...

**Reply by lizheming** [reply]
这个多想想办法吧，加油~

**Reply by MrZhang365** [reply]
啊这……

**Comment by fcjz** [comment]
建议v2ray，节点GitHub。

**Reply by MrZhang365** [reply]
关键是我不会翻墙……

**Reply by fcjz** [reply]
所以我推荐你用v2ray翻墙啊……节点GitHub可以搜到。

**Reply by MrZhang365** [reply]
关键是这是违法的，而且我也没那个胆量。。。

**Reply by wmz1024** [reply]
> 关键是这是违法的，而且我也没那个胆量。。。

那你怎么访问的Github hh

**Reply by fcjz** [reply]
> > 关键是这是违法的，而且我也没那个胆量。。。
> 
> 那你怎么访问的Github hh

如果入GitHub比较早的话，当时GitHub还没被限制吧，可以轻松下载到各种加速工具比如fastgithub，访问速度很快。

**Reply by Muska-Ami** [reply]
> > 关键是这是违法的，而且我也没那个胆量。。。
> 
> 那你怎么访问的Github hh

github又不是完全被屏蔽了，我这里多刷新几次就能正常访问
